data-hub/src/main/resources/ml-modules/root/com.marklogic.hub/lib/flow-lib.xqy

```diff
@@ -369,7 +383,12 @@ declare function flow:run-collector(
   $options as map:map) as xs:string*
 {
   let $module-name := hul:get-module-name($module-uri)
-  let $ns := $PLUGIN-NS || fn:lower-case($module-name)
+  let $filename as xs:string := hul:get-file-from-uri($module-uri)
+  let $type := flow:get-type($filename)
+  let $ns :=
+    if ($type eq "sjs") then ()
```

Comment:
You're using enums in other places; make sense to use them here too for consistency and avoiding magic strings? I've seen one case of someone updating the mime-type list so that ".js" would be treated the same as ".sjs".